tools/test.py

```diff
@@ -136,10 +136,10 @@ def main():
                 with_sync=with_sync,
                 file=args.log2file):
             outputs = task_processor.single_gpu_test(model, data_loader,
-                                                     args.show, args.show_dir)
+                                                     args.show, args.show-dir)
```

Comment:
`show-dir` would be treated as a "sub" expression instead of a variable. I think this is not expected.